src/dispatch/plugins/dispatch_jira/plugin.py

```diff
@@ -76,20 +76,23 @@ def get_user_field(client: JIRA, hosting_type: str, jira_username: str, user_ema
 
         # we default to the Jira user we use for managing issues
         # if we can't find the user in Jira
-        return {"name": jira_username}
-    if hosting_type == "cloud":
-        username = get_email_username(user_email)
-        user = next(
-            client._fetch_pages(
-                User,
-                None,
-                "user/search",
-                startAt=0,
-                maxResults=1,
-                params={"query": username},
-            )
-        )
-        return {"id": user.accountId}
+        return {"name": configuration.jira_username}
+    if configuration.hosting_type == "cloud":
+        import json
+        import requests
```

Comment:
Using `requests` to workaround compatibility issues between Python client version and hosting types.